src/Editor/index.js

```diff
@@ -367,9 +367,11 @@ export default class WysiwygEditor extends Component {
   };
 
   handlePastedText = (text, html) => {
-    if (this.props.handlePastedText &&
-      this.props.handlePastedText(text, html, editorState, this.onChange)) {
-        return true;
+    if (this.props.handlePastedText){
+      const isHandled = this.props.handlePastedText(text, html, editorState, this.onChange);
+      if(this.props.stripPastedStyles || isHandled){
```

Comment:
This will cause paste to never work if `this.props.stripPastedStyles` is true. This is not correct I think.